modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -27,7 +27,7 @@ When you deploy message flows, your existing message flows continue uninterrupte
 
 * Apply environment settings
 +
-Updates all the Mule applications in your runtime with the latest available template versions.
+Updates the Mule applications in your runtime with the latest available template versions.
```

Comment:
Please revert back to the statement where we mention "all the Mule applications".
"Updates all the Mule applications in your runtime with the latest available template versions."